conf/app.routes

```diff
@@ -209,3 +209,13 @@ GET         /update-record/:recordId/check-your-answers/trader-reference
 POST        /update-record/:recordId/check-your-answers/trader-reference                    controllers.CyaUpdateRecordController.onSubmitTraderReference(recordId: String)
 GET         /update-record/:recordId/check-your-answers/commodity-code                      controllers.CyaUpdateRecordController.onPageLoadCommodityCode(recordId: String)
 POST        /update-record/:recordId/check-your-answers/commodity-code                      controllers.CyaUpdateRecordController.onSubmitCommodityCode(recordId: String)
+
+GET        /update-profile/removing-nirms-number                                            controllers.HasNirmsChangeController.onPageLoad(mode: Mode = NormalMode)
+POST       /update-profile/removing-nirms-number                                            controllers.HasNirmsChangeController.onSubmit(mode: Mode = NormalMode)
+GET        /changeHasNirmsChange                                                            controllers.HasNirmsChangeController.onPageLoad(mode: Mode = CheckMode)
```

Comment:
in both cases we don't need the CheckMode version of these routes as we never view them from a CYA perspective we always go straight through the journey, so I would remove the CheckMode routes and then also remove the 'mode' parameters for both of the controllers as they are not needed